---
bool success
string message
navigation_yeon/AllWaypoint[] all_waypoints
